report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,191 @@
 \section{Methodology}\label{sec:methodology}
-Method / Our Contribution (Detailed main part of the story, or how the analysis was done)
+Since we did not have access to the original source code implementing the pipeline, we have replicated it at accurately as we could based on the available information in \citeauthor{andersonImprovedAccuracyQuantitative2017} and \citeauthor{cleggRecalibrationMarsScience2017}.
+However, these papers did not include all the details of the pipeline, so we have had to make some assumptions and creative decisions ourselves.
+In addition, some aspects of the pipeline rely on qualitative assessments made by the original authors --- something we cannot do because we are not domain experts.
+Consequently, our pipeline is not identical to the original, but we have strived to make it as close as possible.
 
-\textit{Recreate the current model. Reproduce the results of the original papers. Investigate the limitations of the model and propose improvements.}
+This section is dedicated to describing the methodology of our pipeline, how it differs from the original, and which design choices we have made and why.
+Furthermore, we delve into the experiments we have conducted to evaluate the performance of our pipeline such that we can identify the components that contribute the most to the overall error.
 
+\subsection{PLS1-SM}
+This section describes the PLS1-SM aspect of the pipeline.
+
+\subsubsection{Outlier Removal with Mahalanobis Distance and Chi-Squared Test}
+% General programming language differences and implementation details
+% Differences in outlier detection (automatic/quantitative vs manual/qualitative)
+% Missing information about how three of the oxides are weighted in MOC result
+% No manual cross validation in PLS
+% Holdout split 80/20
+% No inverse IRF weighing weighting in ICA
+% Missing information about outlier removal in ICA - therefore we decided to not do it
+% Missing information about how ICA uses each of the five datasets for each target
+% How we do regression in ICA (since there are not enough details about it in the papers)
+
+As mentioned in section \ref{sec:background}, if the Mahalanobis distances can be shown to follow a chi-squared distribution, then the chi-squared test can be used to determine whether a sample is an outlier.
+Since we do not have the expertize to make a qualitative assessment of the outliers, we have instead decided to use this property of the Mahalanobis distances to automatically detect outliers.
```

Comment:
```suggestion
Since we do not have the expertise to make a qualitative assessment of the outliers, we have instead decided to use this property of the Mahalanobis distances to automatically detect outliers.
```